build-tools/packages/build-tools/src/fluidBuild/tasks/taskFactory.ts

```diff
@@ -69,36 +70,47 @@ const executableToLeafTask: {
 } as const;
 
 /**
- * Given a command executable, attempts to find a matching `TaskHandler` that will handle the task. If one is found, it
- * is returned; otherwise, it returns `UnknownLeafTask` as the default handler.
+ * Create a leaf task for the given command.
+ * If the task has file dependencies specified, or the executable is a declarative task create a declarative task.
+ * Otherwise, find a known executable task handler for the executable, or use the UnknownLeafTask as a fallback.
  *
- * Any DeclarativeTasks that are defined in the fluid-build config are checked first, followed by the built-in
- * executableToLeafTask constant.
- *
- * @param executable The command executable to find a matching task handler for.
+ * @param node build package for the target task
+ * @param command the command to create the task for
+ * @param context the build context
+ * @param taskName target name
+ * @param files file dependencies for the task, if any
```

Comment:
Are these paths relative to the package or to the repo root? Or are they absolute?

Reply:
Package relative